### DISCUSSION TITLE
Don't disable the Overview dashboard when setting another dashboard as default

### DISCUSSION THREAD

**Discussion by Fabenissimo** [home-assistant/frontend#11246] [Other feature requests]
## The request

Since 107.7 users can add multiple lovelace-dashboards to the UI. I love the feature but I don't get one thing:
Why is the "Overwiew" Dashboard getting disabled when you set another Dashboard as a default on a specific device?
Example: My dad has a seperate Dashboard for all the stuff he uses most of the time, but sometimes he needs to switch to a different Dashboard if his simplified Dashbaord doesn't have the button/card he needs. But If I set his simplified Dashbaord (which he uses 95% of the time) as default on his device, he no longer can access the "Overview" Dashboard where the button/card is which he needs.

## The alternatives

Create a new Dashboard that is a exact copy of the "Overview" Dashboard, with the drawback of having to manage two identical dashbaords.

## Additional information
Other Dashboards don't get disabled when setting another Dashboard as default.

**Comment by dotMorten** [comment]
This has annoyed me a lot as well. I find the default dashboard quite useful, but the custom dashboards I would be using most of the time and want as default.

**Comment by wojciechczyz** [comment]
I would like to be able to choose between what you suggest and the current behaviour per user.

**Comment by dotMorten** [comment]
@wojciechczyz Just turn if off in the sidebar?
